packages/google_maps_flutter/google_maps_flutter_ios/ios/Classes/GoogleMapMarkerController.m

```diff
@@ -225,163 +332,220 @@ @interface FLTMarkersController ()
 
 @property(strong, nonatomic) NSMutableDictionary *markerIdentifierToController;
 @property(strong, nonatomic) FlutterMethodChannel *methodChannel;
+@property(strong, nonatomic) UIImage *emptyClusterMarker;
+@property(strong, nonatomic) NSString *fontPath;
 @property(weak, nonatomic) NSObject<FlutterPluginRegistrar> *registrar;
 @property(weak, nonatomic) GMSMapView *mapView;
 
+
 @end
 
 @implementation FLTMarkersController
 
 - (instancetype)initWithMethodChannel:(FlutterMethodChannel *)methodChannel
                               mapView:(GMSMapView *)mapView
                             registrar:(NSObject<FlutterPluginRegistrar> *)registrar {
-  self = [super init];
-  if (self) {
-    _methodChannel = methodChannel;
-    _mapView = mapView;
-    _markerIdentifierToController = [[NSMutableDictionary alloc] init];
-    _registrar = registrar;
-  }
-  return self;
+    self = [super init];
+    if (self) {
+        _methodChannel = methodChannel;
+        _mapView = mapView;
+        _markerIdentifierToController = [[NSMutableDictionary alloc] init];
+        _registrar = registrar;
+        _emptyClusterMarker = [self baseClusterMarker];
+        _fontPath = [self loadFont];
+    }
+    return self;
+}
+
+
+void CFSafeRelease(CFTypeRef cf) {
+    if (cf != NULL) {
+        CFRelease(cf);
+    }
+}
+
+-(NSString *)loadFont {
+    NSBundle *frameworkBundle = [NSBundle bundleForClass:self.classForCoder];
+    NSURL *bundleURL = [[frameworkBundle resourceURL] URLByAppendingPathComponent:@"CozyFonts.bundle"];
+    NSBundle *bundle = [NSBundle  bundleWithURL:bundleURL];
+    NSURL *fontURL = [bundle URLForResource:@"oatmealpro2_semibold" withExtension:@"otf"];
+    NSData *inData = [NSData dataWithContentsOfURL:fontURL];
+    CFErrorRef error;
+    CGDataProviderRef provider = CGDataProviderCreateWithCFData((CFDataRef)inData);
+    CGFontRef font = CGFontCreateWithDataProvider(provider);
```

Comment:
To make this function smaller and easier to read, WDYT about breaking this method into two functions: a pure function for getting the fontRef and another one for registering it?